### ISSUE TITLE
Starter kit : "could not find hyperbook.json"

### ISSUE THREAD

**Issue by eslachance** [openpatch/hyperbook#863]
Was looking for a gitbook/retype alternative for my docs. 

Tried to download the hyperbook-anywhere zip, installed the Hyperbook VSCode extension, clicked the "Hyperbook" link in the vscode footer (also tried running the command from the command pallette). 

Saw error message, flipped a table, had a nervous breakdown, gave up.

Perhaps this will help someone if y'all can figure out why it happens. 

![image](https://github.com/openpatch/hyperbook/assets/643344/36eca470-bf3b-471a-90eb-f3406327e7f1)
![image](https://github.com/openpatch/hyperbook/assets/643344/dd4d68d8-266e-46e5-bf07-93fd12a89d86)

Version: 1.88.0 (user setup)
Commit: 5c3e652f63e798a5ac2f31ffd0d863669328dc4c
Date: 2024-04-03T13:26:18.741Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by eslachance** [comment]
Additionally, tried to follow "deploy" instructions in case that would work, got this : 

```
PS I:\hyperbook-anywhere-main> npx hyperbook build
Need to install the following packages:
hyperbook@0.29.0
Ok to proceed? (y)
[Hyperbook GitHub Pages] Building Book.
Could not find .hyperbook folder at Could not find .hyperbook folder. Be sure to run `hyperbook setup`.. Be sure to run `hyperbook setup`.
PS I:\hyperbook-anywhere-main> 
```

**Comment by mikebarkmin** [comment]
Regarding the not found `hyperbook.json`: This is likely an issue on Windows. Maybe the paths are not set independent. On my Arch Linux installation, is works. 

Regarding the second issue: You need to run `hyperbook setup` first, but I think it will most likely fail with a similar issue.

**Comment by mikebarkmin** [comment]
Fixed in #867
